packages/usa-modal/src/test/modal-nested-form.spec.js

```diff
@@ -0,0 +1,161 @@
+const assert = require("assert");
```

Comment:
I am curious why a new `spec.js` file and `template` were created for nested forms. The files seem very similar. Would it be possible to just add these updates and the ones from `modal-nested-form.template.html` to `modal.spec.js` and `test/template.html` instead of creating new files?

Reply:
I believe I had a discussion with @mejiaj about the test and we mentioned creating a new test with the nested form. I also referenced combo box which has a different template and test for each of its variants.

Happy to make this the sole test if we prefer to do it all in one sweep!

Reply:
@mejiaj do you have a preference on this?

Reply:
@amyleadem @mahoneycm initially I suggested this as it's own test to keep the template and code simple. It looks like the test template and code are pretty much identical. In that case it makes sense to combine into one.

I'm okay if we simplify the test code to bare minimum or combine them into a single `modal.spec.js`.